apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/request/QuestionValidationRequest.java

```diff
@@ -0,0 +1,5 @@
+package gov.cdc.nbs.questionbank.question.request;
+
+public record QuestionValidationRequest(String field, String value) {
```

Comment:
Thoughts on having an `enum` for the field? This will let the UI know exactly what values are valid here.